eng/Subsets.props

```diff
@@ -84,11 +87,14 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <!-- Global -->
-    <SubsetName Include="RegenerateReadmeTable" Category="" OnDemand="true" Description="Regenerates the table of asset links in the README.md file." />
-
     <!-- Libraries -->
-    <SubsetName Include="All" Category="Libraries" Description="The .NET libraries comprising the shared framework." />
+    <SubsetName Include="Native" Category="Libraries" Description="The native libraries used in the shared framework." />
+    <SubsetName Include="Ref" Category="Libraries" Description="The managed reference libraries." />
+    <SubsetName Include="Src" Category="Libraries" Description="The managed implementation libraries." />
+    <SubsetName Include="ApiCompat" Category="Libraries" Description="The ApiCompat tool which scans for api compatibility." />
```

Comment:
I'm hesitant about this subset, I think it should just be part of the src build.

Reply:
@ericstj thoughts?

Reply:
I was actually going to say the same.  I don't think it needs to be separate.  I was assuming @ViktorHofer separated it because src was parallel but ApiCompat needs to happen after all src libs.  If that can be represented within the src subset then that would be preferable as I don't expect folks to need to run ApiCompat independent of src build.  I think PreTest should be similar.

Reply:
Sounds good. Pretest needs to stay separate as it should be invokable from outside the src build.

Reply:
I think the `PreTest` subset is handy in the sense that some times people need to run `PreTest` to generate the deps.json file again or some sorts like that, so it is better to just do, build.cmd -subset pretest, rather than remembering the project that did it and invoke MSBuild on it.